packages/lib/src/vis/shared/Html.tsx

```diff
@@ -4,13 +4,21 @@ import type { ReactNode } from 'react';
 import ReactDOM from 'react-dom';
 
 interface Props {
+  overflowCanvas?: boolean;
   container?: HTMLElement;
   children?: ReactNode;
 }
 
 function Html(props: Props) {
-  const gl = useThree((state) => state.gl);
-  const { container = gl.domElement.parentElement, children } = props;
+  const {
+    overflowCanvas = false,
+    container: customContainer,
+    children,
+  } = props;
+
+  const r3fRoot = useThree((state) => state.gl.domElement.parentElement);
+  const container =
+    customContainer || (overflowCanvas ? r3fRoot?.parentElement : r3fRoot);
```

Comment:
I think a comment is due here to explain why this change of containers makes the overflow work differently